### ISSUE TITLE
Getting two fields or more as "attribute" in relations

### ISSUE THREAD

**Issue by adbmdp** [Laravel-Backpack/CRUD#348]
Is it possible to concat two fields for the attribute (foreign key attribute that is shown to user) ?
For example i can display the name of the user with the relation below but i would like to display first + name. 

```
$this->crud->addField([
            'label' => "User",
            'type' => 'select2_multiple',
            'name' => 'users',
            'entity' => 'users',
            'attribute' => 'name',
            'model' => "App\Models\User",
            'pivot' => true
]);
```
Something like :
`'attribute' => 'firstname' . 'name',`

Thanks.

**Comment by tabacitu** [comment]
Sure, just create [an accesor](https://laravel.com/docs/5.3/eloquent-mutators#accessors-and-mutators) on your User model, something like:

```php
public function getFullNameAttribute($value) {
   return $this->firstname.' '.$this->name;
}
```

You'll then be able to use ```full_name``` for the attribute, both in CRUD and anywhere else in your app (```$user->full_name```).

Hope it helps.

Cheers!

**Comment by adbmdp** [comment]
Awesome. Thank you!

**Comment by pxpm** [comment]
Thanks @singeryo it's definitely helpful. And not only for ajax queries, for any scenario where you want to add something else to the model you should use Laravel model `$appends` . 
Cheers

**Comment by kushsha** [comment]
#define into user model
 public function getFullNameAttribute($value) {
        return $this->first_name.' '.$this->last_name;
    }

#use in controller
$this->crud->addcolumn([
            // role list
            'label' => 'Primary program manager',
            'name' => 'priorityProgramManger.full_name', // the method that defines the relationship in your Model
        ]);

**Comment by altayevrim** [comment]
This is also helpful when you want to use relations with Ajax. 

```php
# on Model
protected $appends = [
    'full_title'
];

public function getFullTitleAttribute():string
{
    return $this->company_number . ' - ' . $this->title;
}
```

```php
# on CrudController
CRUD::field('company')->type('relationship')->label('Şirket')->attribute('full_title')->ajax(true);
```

In case someone (or me) needs it for later.
